src/conversation.ts

```diff
@@ -242,7 +242,7 @@ class ConversationControls {
      */
     public async exit(id?: string) {
         const session = await this[internal].session();
-        if (session.conversation === undefined) return;
+        if (session.conversation == undefined) return;
```

Comment:
Can you change this?
```suggestion
        if (session.conversation == null) return;
```
This would be consistent with the rest of the code in the grammY ecosystem, and it makes it a bit easier to spot the coercion because null comparisons are so uncommon.